### ISSUE TITLE
Usage of deprecated API of SQLAlchemy - FromClause.count

### ISSUE THREAD

**Issue by ayalash** [getlogbook/logbook#300]
```
tests/test_ticketing.py::test_basic_ticketing
  /home/ayalas/src/github/logbook/logbook/ticketing.py:246: SADeprecationWarning: ``FromClause.count()`` is deprecated. Counting rows requires that the correct column expression and accommodations for joins, DISTINCT, etc. must be made, otherwise results may not be what's expected. Please use an appropriate ``func.count()`` expression directly.
    return self.engine.execute(self.tickets.count()).fetchone()[0]
```

**Comment by ayalash** [comment]
Relevant SQLAlchemy code:
https://github.com/sqlalchemy/sqlalchemy/blob/0e9da6ee8d54bd0cde9ef3fb1ce3bceca841624f/lib/sqlalchemy/sql/selectable.py#L339-L366

**Comment by RazerM** [comment]
Resolved by #331
